app/assets/javascripts/config/angularRoutes.js.coffee

```diff
@@ -669,6 +669,27 @@
         ]
     })
     # Short form: "You" section
+    .state('dahlia.short-form-application.prerequisites', {
+      url: '/prerequisites'
+      views:
+        'container':
+          templateUrl: 'short-form/templates/b0a-prerequisites.html'
+      params:
+        infoChanged:
+          squash: true
+      onEnter: [
+        '$stateParams', 'ShortFormApplicationService', 'AccountService', 'AutosaveService'
+        ($stateParams, ShortFormApplicationService, AccountService, AutosaveService) ->
+          ShortFormApplicationService.completeSection('Intro')
+          if AccountService.loggedIn()
+            ShortFormApplicationService.importUserData(AccountService.loggedInUser)
+            ShortFormApplicationService.infoChanged = $stateParams.infoChanged
+            # always autosave when you start a new application
+            # TODO: remove hotfix for marking initial autosaves that come from the Name page
+            unless ShortFormApplicationService.application.id
+              ShortFormApplicationService.submitApplication({autosave: true, initialSave: true})
+      ]
```

Comment:
@callaghanc I'm not sure if all of those things here are needed. Could you have a look?

Reply:
@software-project this onEnter block makes sense. for ownership applications, this is the first page of the application where the user begins to enter information. so, marking the Intro section as complete makes sense because the intro is over at this point and the user is now filling out the app. the autosave also could make sense, because the current intention with autosave is to save the user's application when they start it. however, it's a little redundant with the autosave on the Name page. for ownership applications, this will make us do two autosaves. we'll still just be doing one for rental apps. this isn't a big problem, but it would be a little nicer to only have one autosave-at-start per app. so we could:

a) add some logic to not autosave on the Name page if we already autosaved on the Prerequisites page, or to not autosave on the Name page if it's an ownership application, or
b) remove the autosave from the Prerequisites page and only autosave on the Name page.

keeping autosave on the Name page only is less complicated, but we'd have to decide whether we are fine with not autosaving at the technical first-filled-out page of the ownership app. @akegan do you have a preference from a business rules/UX perspective?

Reply:
It's a really interesting question because I believe autosave is actually totally disabled for the entire app currently, and we already had to disable it further in (this ticket)[https://www.pivotaltracker.com/story/show/159252180].

In terms of the logic of where to save if we have it saving at the start of the app, I'm not sure. If it's just the checkboxes, then the user isn't really losing anything if it's not saved, but if they've uploaded multiple documents then that's significant time already spent.

tl;dr, I'm not sure! I can follow up with josh

Reply:
I think we actually disabled it completely in this PR https://github.com/Exygy/sf-dahlia-web/commit/0a1b65a8609db124cc342cff0ed38ea80ee4757b#diff-bf948082a3b5ee08694d1e3427c28a7fL721

Reply:
hmm then do we want to keep an autosave call in any onEnters at all? what's the roadmap looking like for re-introducing use of autosave?

Reply:
let's just leave it on the name page for now. It's not on the roadmap at all yet, but my hope is that it might be possible once 8cloud comes on board and starts working on making the API better

Reply:
👍